### PR TITLE
Resolve of Issue #31.

### PR DESCRIPTION
**Pull request by Artakarak**
**Summary**
We now can attain a score within our RoundView and LoadableGamesView. Functionality seems to be good as score is updated and shown properly within a round, and persisted just as well with `LoadableGamesView`.

**Visuals**
Image 2023-11-29 at 3 55 PM (1)

----

Image 2023-11-29 at 3 55 PM

### CODE REVIEW

File: test/use_case/get_loadable_games/load_game/LoadGameInteractorTest.java [hunk 1]
```diff
@@ -1,9 +1,10 @@
-package use_case.load_game;
+package use_case.get_loadable_games.load_game;
```
Comment (rayceramsay) [inline test/use_case/get_loadable_games/load_game/LoadGameInteractorTest.java:1]: It looks like you accidentally moved the load_game folder in test/use_case to be under get_loadable_games. If you could move it back to be at the same level as the others that would be great.

**Review by rayceramsay — CHANGES_REQUESTED** [review]
Nice job Artak! Good work on updating the tests as well, those are easy to forget about. Just a couple of notes:

It would appear that the score label displays the same score as the previously played game for the first round when you create a new game (e.g. if I have a score in the previous game of 2 then go to the menu and create a new game, it will say I have a score of 2). This will require a minor fix with updating the RoundState on game creation to set the score to the proper value (i.e. 0).

Going forward, making your commit messages a bit more descriptive would help those reviewing your PRs. It doesn't have to be anything crazy, but for example, I don't really know what `Final touches, fully done.` is referencing. If you said something like `Added score label to RoundView` instead, then I would have a better idea of what the commit was for. This is especially useful for maintenance in the long run as it makes it easier to figure out where bugs come from if they arise.

**Comment by Artakarak** [comment]
> Nice job Artak! Good work on updating the tests as well, those are easy to forget about. Just a couple of notes:
> 
> It would appear that the score label displays the same score as the previously played game for the first round when you create a new game (e.g. if I have a score in the previous game of 2 then go to the menu and create a new game, it will say I have a score of 2). This will require a minor fix consisting of updating the RoundState on game creation to set the score to the proper value (i.e. 0).
> 
> Going forward, making your commit messages a bit more descriptive would help those reviewing your PRs. It doesn't have to be anything crazy, but for example, I don't really know what `Final touches, fully done.` is referencing. If you said something like `Added score label to RoundView` instead, then I would have a better idea of what the commit was for. This is especially useful for maintenance in the long run as it makes it easier to figure out where bugs come from if they arise.

Yeah Rayce that's my bad. Thank you for making me 1% better today. I have fixed the remaining error that you mentioned, let me know of anything else that you may catch!

**Review by rayceramsay — APPROVED** [review]
Works very well! 🙂

**Review by NickinAction — APPROVED** [review]
Looks good and works well!

**Pull request merged**